odl/space/pspace.py

```diff
@@ -873,24 +873,42 @@ def show(self, title=None, indices=None, **kwargs):
             else:
                 indices = list(np.linspace(0, self.size - 1, 4, dtype=int))
         else:
-            if isinstance(indices, tuple):
+            if (isinstance(indices, tuple) or
+                    (isinstance(indices, list) and
+                     not all(isinstance(idx, Integral) for idx in indices))):
+                # Tuples or lists containing non-integers index by axis.
+                # We use the first index for the current pspace and pass
+                # on the rest.
                 indices, kwargs['indices'] = indices[0], indices[1:]
 
+            # Support `indices=[None, 0, None]` like syntax
```

Comment:
Actually yes, but it was obscured a bit by the code above it since it changed what `indices` is.